### PR TITLE
Add instructions for setting up OIDC/RBAC for the dashboard

### PR DESCRIPTION
**Pull request by yiannistri**
Add instructions for setting up OIDC/RBAC for the dashboard

### CODE REVIEW

File: docs/gitops-dashboard.md [hunk 1]
```diff
@@ -27,5 +27,100 @@ Applications being managed by Weave GitOps are displayed in a list. Clicking the
...
+
+```yaml title="apps-writer.yaml"
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: apps-writer
+rules:
+- apiGroups: ["apiextensions.k8s.io"]
+  resources: ["customresourcedefinitions"]
+  verbs: ["get"]
```
Comment (palemtnrider) [inline docs/gitops-dashboard.md:107]: Is this the complete list of verbs needed for the writer?

File: docs/gitops-dashboard.md [hunk 2]
```diff
@@ -27,5 +27,100 @@ Applications being managed by Weave GitOps are displayed in a list. Clicking the
...
+
+### Configuration
+
+After enabling the feature, `gitops ui run` will require the following additional parameters:
+
+| Parameter                     | Type | Description                                                           | Default   |
+| ----------------------------- | - |--------------------------------------------------------------------- | --------- |
+| `--oidc-issuer-url`           | string | The URL of the issuer, typically the discovery URL without a path     |           |
+| `--oidc-issuer-client-id`     | string | The client ID that has been setup for Weave GitOps in the issuer      |           |
+| `--oidc-issuer-client-secret` | string | The client secret that has been setup for Weave GitOps in the issuer  |           |
```
Comment (palemtnrider) [inline docs/gitops-dashboard.md:53]:
```suggestion
| `--oidc-client-id`     | string | The client ID that has been setup for Weave GitOps in the issuer      |           |
| `--oidc-client-secret` | string | The client secret that has been setup for Weave GitOps in the issuer  |           |
```

**Review by palemtnrider — APPROVED** [review]
LGTM

File: docs/gitops-dashboard.md (hunk 1 above)
Comment (yiannistri) [reply docs/gitops-dashboard.md:107]: Yes, because everything else is handled via flux

**Pull request merged**